src/main/java/project/cinema/model/dto/OrderResponseDto.java

```diff
@@ -0,0 +1,43 @@
+package project.cinema.model.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketsIdList;
+    private LocalDateTime orderDate;
```

Comment:
```suggestion
    private String orderDate;
```

Reply:
Bohdan said that better to send LocalDateTime 
![image](https://user-images.githubusercontent.com/51205960/108092249-2f369680-7085-11eb-80b1-66e702fa446f.png)